source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -1212,7 +1233,7 @@ void Graph::buildUiBaseGraph(mx::DocumentPtr doc)
         std::string name = nodeGraph->getName();
         auto currNode = std::make_shared<UiNode>(name, _graphTotalSize);
         currNode->setNodeGraph(nodeGraph);
-        setUiNodeInfo(currNode, "", "nodegraph");
+        setUiNodeInfo(currNode, "", mx::Implementation::NODE_GRAPH_ATTRIBUTE);
```

Comment:
There a few hard-coded string replacements with constants to avoid string initialization overhead which is a niggling performance slowdown. I didn't do all of them :) ).